packages/models/src/lib/upload-config.ts

```diff
@@ -1,18 +1,14 @@
 import { z } from 'zod';
-import { urlSchema } from './implementation/schemas';
+import { slugSchema, urlSchema } from './implementation/schemas';
 
 export const uploadConfigSchema = z.object({
   server: urlSchema('URL of deployed portal API'),
   apiKey: z.string({
     description:
       'API key with write access to portal (use `process.env` for security)',
   }),
-  organization: z.string({
-    description: 'Organization in code versioning system',
-  }),
-  project: z.string({
-    description: 'Project in code versioning system',
-  }),
+  organization: slugSchema('Organization in code versioning system'),
+  project: slugSchema('Project in code versioning system'),
```

Comment:
I notice the descriptions aren't very accurate :sweat_smile: 

```suggestion
  organization: slugSchema('Organization slug from Code PushUp portal'),
  project: slugSchema('Project slug from Code PushUp portal'),
```

Reply:
Good point, fixed.